### ISSUE TITLE
Ugly outline, also appears when pressing a key other than Tab

### ISSUE THREAD

**Issue by Linerly** [vector-im/element-web#19889]
### Steps to reproduce

1. Press a key (like Esc, Meta (Windows key), Tab, Shift, ...)
2. See an ugly yellow-white outline
![1](https://user-images.githubusercontent.com/47475676/143377294-69e603ab-b158-4375-848f-14c59b9b57ec.png)

#### What did you expect?
Pressing keys other than Tab shouldn't show the outline, the outline should be like this, which isn't ugly:
![2](https://user-images.githubusercontent.com/47475676/143377349-4178d98d-6ec5-490c-a8de-021ca413261d.png)

…or like Discord's:
![3](https://user-images.githubusercontent.com/47475676/143377739-d1fe843e-25a9-4ff1-8cfa-30df856b91c3.png)

#### What happened instead?
The outline shows whenever I press a key other than Tab (e.g., when taking a screenshot using the shortcut Meta + Shift + PrtSc)

### Operating system

Arch Linux

### Application version

Element Nightly version: 2021112401 Olm version: 3.2.3

### How did you install the app?

AUR

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by t3chguy** [comment]
> See an ugly yellow-white outline

This is your browser's accessibility outline ftr, in Firefox it looks very different

**Comment by SimonBrandner** [comment]
> > See an ugly yellow-white outline
> 
> This is your browser's (Element Desktop is Electron thus Chromium) accessibility outline ftr, in Firefox it looks very different

Yeah, though I believe there is a way to change the look of this since Discord has the blueish one on desktop

**Comment by t3chguy** [comment]
Yes we can definitely style it, I'm not saying we can't, just explaining what it is better than "ugly outline" - not that it isn't also that

**Comment by Penguin-Guru** [comment]
I think The Super/Meta/etc... button is particularly important to disconnect from this functionality. Convention on all platforms (I know of) has long considered this a system utility button, not to be used by applications. Therefore, a lot of common keyboard shortcuts use this button.

For what it's worth, I use Chromium as my normal web browser and this accessibility outline has never appeared when I press these buttons on my keyboard. It may be useful for some people, I don't know, but I definitely think it should disabled by default in the Element desktop client.

**Comment by HarHarLinks** [comment]
I don't really mind it, but

> Convention on all platforms (I know of) has long considered this a system utility button, not to be used by applications. Therefore, a lot of common keyboard shortcuts use this button.

I agree with this, and want to point out that this behaviour might be confusing for users less familiar with the existence of such features.
